runtime/queries/php-only/injections.scm

```diff
@@ -0,0 +1,10 @@
+((comment) @injection.content
+  (#set! injection.language "phpdoc"))
```

Comment:
```suggestion
  (#set! injection.language "comment"))
```

We don't currently have a `phpdoc` language. We can use `comment` which has some basic highlighting for `TODO`, `HACK`, etc.